### PR TITLE
1 add landing message

### PR DESCRIPTION
**Pull request by CelestialLamb**
please pull this code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,11 +4,12 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Potionsss</title>
+    <title>Potionsss test change</title>
```
Comment (IrFoundGitTest2) [inline index.html:7]: You seem to have left the word test in this code??

**Review by IrFoundGitTest2 — CHANGES_REQUESTED** [review]
Just fix the title by removing the 'test' string, and maybe make the heading red?

**Comment by CelestialLamb** [comment]
Of course!

**Review by IrFoundGitTest2 — COMMENTED** [review]
please change the code as per my requests.

**Pull request closed without merging**